mlflow/deployments/cli.py

```diff
@@ -0,0 +1,154 @@
+import click
+from mlflow.utils import cli_args
+from mlflow.deployments import interface
+
+
+def _user_args_to_dict(user_list):
+    # Similar function in mlflow.cli is throwing exception on import
+    user_dict = {}
+    for s in user_list:
+        try:
+            name, value = s.split('=')
+        except ValueError:
+            # not enough values to unpack
+            raise click.BadOptionUsage("config", "Config options must be a pair and should be"
+                                                 "provided as ``--C key=value`` or "
+                                                 "``--config key=value``")
+        if name in user_dict:
+            raise click.ClickException("Repeated parameter: '{}'".format(name))
+        user_dict[name] = value
+    return user_dict
+
+
+target_uri = click.option("--target-uri", "t_uri", required=True,
+                          help="Deployment target URI. Check the documentation/help for each "
+                               "plugin to understand the target URI format the plugins expect. "
+                               "For fetching the help, you can call "
+                               "`mlflow deployments help --target-name <target-name>`")
+target_name = click.option("--target-name", "t_name", required=True,
+                          help="Deployment target name.")
+deployment_name = click.option("--name", "name", required=True,
+                               help="Name of the deployment")
+parse_custom_arguments = click.option("--config", "-C", metavar="NAME=VALUE", multiple=True,
+                                      help="Extra target-specific config for the model deployment,"
+                                           " of the form -C name=value. See documentation/help for "
+                                           "your deployment target for a list of supported config"
+                                           " options.")
+context_settings = dict(allow_extra_args=True, ignore_unknown_options=True,)
```

Comment:
I think we can remove this, as we don't need to allow custom CLI arguments to any of the commands anymore, right?